packages/timeline-state-resolver/src/integrations/httpSend/index.ts

```diff
@@ -366,10 +246,18 @@ export class HTTPSendDevice extends DeviceWithState<HTTPSendState, DeviceOptions
 					'EAI_AGAIN',
 				]
 
-				if (retryCodes.includes(err.code) && this._resendTime) {
-					const timeLeft = Math.max(this._resendTime - (Date.now() - t), 0)
-					await new Promise<void>((resolve) => setTimeout(() => resolve(), timeLeft))
-					this._defaultCommandReceiver(_time, cmd, context, timelineObjId, layer).catch(() => null) // errors will be emitted
+				if (retryCodes.includes(err.code) && this.options?.resendTime) {
+					const timeLeft = Math.max(this.options.resendTime - (Date.now() - t), 0)
+					setTimeout(() => {
```

Comment:
This probably isn't a new bug, but this retry will keep going after the device is disposed.
This could be solved by a simple check eitehr inside this `setTimeout`, or at the start of `sendCommand`, to return early if the device has been disposed

Reply:
Yeah, there is also a problem where commands sent through tsr actions will be resent forever. I think I'll add something quick n dirty for both edge cases.